src/components/DiagramMoney.tsx

```diff
@@ -1,18 +1,75 @@
 import * as React from 'react';
 import { useEffect, useState } from 'react';
+
 import money from '../assets/stats/money.png';
-import { Input, Text } from '@chakra-ui/react';
+import { Text, Flex, Center, Box, VStack, HStack } from '@chakra-ui/react';
+import { Bar, BarChart, XAxis, YAxis } from 'recharts';
+import dataJson from '../assets/MockData.json';
 
 const DiagramMoney = () => {
     const [count, setCount] = useState<string>('');
 
     useEffect(() => {}, []);
 
+    function moneyToList() {
+        const moneyList: number[] = [];
+        dataJson.EnergyMoneyMonthly.forEach((month) =>
+            moneyList.push(month.spent)
+        );
+        return moneyList;
```

Comment:
Her kan du bruke map funksjonen, typ: `const moneyToList = () => data.Json.EnergyMoneyMonthly.map((month) => month.spent);`

Reply:
@asewilhelmsen du får se på de foreslåtte endringene :) Bare å endre i statsMoney for er allerede gjort noen endringer der som vi brukte under brukertestene så må merges på nytt uansett